toolkit/scss/_notification-banners.scss

```diff
@@ -1,6 +1,10 @@
 @import "_colours.scss";
 @import "_digital-marketplace-colours.scss";
 @import "_typography.scss";
+@import "_measurements.scss";
+
+// Required placeholders:
+// @import "placeholders/_temporary-messages.scss";
```

Comment:
This reads like an import that’s been commented out. Might read better without the `@import`?